core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -165,7 +189,8 @@ object ConsumerPerformance {
 
         if (currentTimeMillis - lastReportTime >= config.reportingInterval) {
           if (config.showDetailedStats)
-            printProgressMessage(0, bytesRead, lastBytesRead, messagesRead, lastMessagesRead, lastReportTime, currentTimeMillis, config.dateFormat)
+            printProgressMessage(0, bytesRead, lastBytesRead, messagesRead, lastMessagesRead,
+              lastReportTime, currentTimeMillis, config.dateFormat, Some(testStartTime), Some(joinTime.get))
```

Comment:
Since `joinTime` is an AtomicLong, and `printProgressMessage` expects an Option, that's why I used `Some(joinTime.get)` here. 
For old consumers, there is no join time, so `printProgressMessage` exposes an Option here. Does it make any senses?